Makefile

```diff
@@ -121,7 +121,7 @@ macos: check
 	ls -la ./build/pup/
 	ls -la ./dist/
 
-linux: check
```

Comment:
Temporarily removing running the tests, as it currently fails due to this unrelated issue: https://github.com/mu-editor/mu/issues/2091

Reply:
After PR https://github.com/mu-editor/mu/pull/2238 was merged and this PR rebased I was able to remove this workaround.